src/app/utils/testing.utils.ts

```diff
@@ -0,0 +1,54 @@
+import { FormGroupDirective, FormGroup } from '@angular/forms';
+import { Applicant, Microfilm } from '@app/models/tech-record.model';
+import { Plate } from '../models/tech-record.model';
+
+/**
+ * *******************SOME GROUND RULES*************************************
+ *
+ *
+ * We use this util file essentially for unit testing to enable a clean mock object provision
+ * for our unit testing. Ideally more simplified object should/can be added.
+ * Amendments of returned object should be done less often or not at ALL. We use
+ * partial<T> object implementation hence the required object update(s) should be
+ * passed in from the specific unit test of interest.
+ *
+ */
+
+export const TESTING_UTILS = {
+    mockApplicant,
+    mockMicrofilm,
+    mockPlates,
+  };
+
+function mockApplicant(args?: Partial<Applicant>): Applicant {
+    const mock: Applicant = {
+      name: 'Ben',
+      address1: 'Robert green',
+      address2: 'Birmingham',
+      postTown: 'lala land',
+      address3: 'MyCounty',
+      postCode: 'AC 123',
+      emailAddress: 'test@test.com',
+      telephoneNumber: '0207777777'
+    };
+    return { ...mock, ...args };
+  }
+
+  function mockMicrofilm(args?: Partial<Microfilm>): Microfilm {
+    const mock: Microfilm = {
+      microfilmDocumentType: 'PSV Miscellaneous',
+      microfilmRollNumber: '1234',
+      microfilmSerialNumber: '5678'
+    };
+    return { ...mock, ...args };
+  }
+
+  function mockPlates(args?: Partial<Plate[]>): Plate[] {
```

Comment:
So this should be one object at a time. 

function mockPlates(args?: Partial<Plate>): Plate[] {
    const mock: Plate= {
        plateSerialNumber: '123123',
        plateIssueDate: '2019-12-13',
        plateReasonForIssue: '5678',
        plateIssuer: 'Issuer1'
    };
    return { ...mock, ...args };
  }